allensdk/brain_observatory/behavior/session_apis/data_io/behavior_ophys_lims_api.py

```diff
@@ -0,0 +1,164 @@
+import logging
+from typing import Optional
+import pandas as pd
+
+from allensdk.api.cache import memoize
+from allensdk.brain_observatory.behavior.session_apis.data_io.ophys_lims_api \
+    import OphysLimsApi
+from allensdk.brain_observatory.behavior.session_apis.data_io import (
+    BehaviorLimsApi)
+from allensdk.internal.api import db_connection_creator, PostgresQueryMixin
+from allensdk.internal.core.lims_utilities import safe_system_path
+from allensdk.core.auth_config import (
+    LIMS_DB_CREDENTIAL_MAP, MTRAIN_DB_CREDENTIAL_MAP)
+from allensdk.core.authentication import credential_injector, DbCredentials
+from allensdk.brain_observatory.behavior.session_apis.data_transforms import (
+    BehaviorOphysDataXforms)
+
+
+class BehaviorOphysLimsApi(BehaviorOphysDataXforms,  OphysLimsApi,
```

Comment:
In general, for your classes, you could be more consistent in the class-level docstring, and I'd err on the side of more descriptive of the use case rather than, for example:
`         This class is used by both Scientifica and Mesoscope ophys experiments.`

Reply:
FYI @djkapner some of these nits are probably from old code.

Reply:
Added more descriptive docstrings.